docs/docs/launch-manual-v2/sections/authoring-packages/sections/publishing-a-package.md

```diff
@@ -0,0 +1,71 @@
+## Publishing a Package
+
+Pulsar bundles a command line utility called `ppm` which we first used back in [Command Line](../../../using-pulsar/#command-line) to search for and install packages via the command line. This is invoked by using the `pulsar` command with the `-p` or `--package` option. The `pulsar -p` command can also be used to publish packages to the Pulsar Package Registry and to update them.
+
+See more in the [PPM Docs]() section.
+
+### Prepare Your Package
+
+There are a few important things to double check before publishing your package:
+
+- Your `package.json` file has a `name`, `description`, and `repository` fields.
+- Your `package.json` `name` field is URL Safe, as in it doesn't include an emoji or special character.
+- Your `package.json` `version` field is [Semver V2](https://semver.org/spec/v2.0.0.html) compliant.
+- Your `package.json` `engines` field contains an entry for `atom` such as: `"enginges": { "atom": ">=1.0.0 <2.0.0"}`.
+- Your package has a `README.md` file at it's root.
+- Your `repository` URL in the `package.json` file is the same as the URL of your repository on GitHub.
+- Your package is in a git repository that has been pushed to [GitHub](https://github.com). Follow [this guide](https://help.github.com/articles/importing-a-git-repository-using-the-command-line/) if your package isn't already on GitHub.
+
+### Publishing Your Package
+
+Before you publish a package it is a good idea to check ahead of time if a
+package with the same name has already been published to
+[the Pulsar Package Repository](https://web.pulsar-edit.dev/packages). You can
+do that by visiting `https://web.pulsar-edit.dev/packages/your-package-name` to
+see if the package already exists. If it does, update your package's name to
+something that is available before proceeding.
+
+Now let's review what the `pulsar -p publish` command does:
+
+1. Registers the package name on Pulsar Package Repository if it is being
+   published for the first time.
+2. Updates the `version` field in the `package.json` file and commits it.
```

Comment:
tbh idk what exactly the update is. patch, minor, or major, or is this about the database somehow?

Edit: Oh this is answered below. This isn't really a review since it comes before the information. Maybe replace with a summary, or move after the info?